src/autoscaler/pruner/config/config.go

```diff
@@ -18,87 +18,73 @@ type LoggingConfig struct {
 	Level string `yaml:"level"`
 }
 
-var defaultLoggingConfig = LoggingConfig{
-	Level: DefaultLoggingLevel,
-}
-
-type DbConfig struct {
-	MetricsDbUrl    string `yaml:"metrics_db_url"`
-	AppMetricsDbUrl string `yaml:"app_metrics_db_url"`
-}
-
-type PrunerConfig struct {
-	MetricsDbPruner    MetricsDbPrunerConfig    `yaml:"metrics_db"`
-	AppMetricsDbPruner AppMetricsDbPrunerConfig `yaml:"app_metrics_db"`
-}
-
 type MetricsDbPrunerConfig struct {
-	RefreshIntervalInHours int `yaml:"refresh_interval_in_hours"`
-	CutoffDays             int `yaml:"cutoff_days"`
+	DbUrl                  string `yaml:"db_url"`
+	RefreshIntervalInHours int    `yaml:"refresh_interval_in_hours"`
+	CutoffDays             int    `yaml:"cutoff_days"`
 }
 
 type AppMetricsDbPrunerConfig struct {
-	RefreshIntervalInHours int `yaml:"refresh_interval_in_hours"`
-	CutoffDays             int `yaml:"cutoff_days"`
+	DbUrl                  string `yaml:"db_url"`
+	RefreshIntervalInHours int    `yaml:"refresh_interval_in_hours"`
```

Comment:
Use Duration and one can specify 24h if we want.
Switch to goyaml instead of candiedyaml.